src/main/java/com/google/devtools/build/lib/rules/cpp/CppHelper.java

```diff
@@ -885,6 +886,23 @@ public static Artifact createDefFileActions(
     return defFile;
   }
 
+  /**
+   * Create action for generating trivial DEF file without any exports, should only be used when
```

Comment:
"Trivial" carries little meaning for me. Can we find a more precise name? What do you think about "no-export" or "fallback"?

And how about calling the function `createDefFileForDllWithoutExports` or something similar?

Reply:
I replaced it with "empty" here, because it's essentially an empty file. Does it sound good?